internal/controller/volumereplicationgroup_controller.go

```diff
@@ -694,6 +714,54 @@ func (v *VRGInstance) updatePVCList() error {
 	return v.separatePVCsUsingStorageClassProvisioner(pvcList)
 }
 
+func (v *VRGInstance) updatePVCListForCG() error {
+	if v.instance.Spec.Async == nil {
+		return nil
+	}
+
+	if !rmnutil.IsCGEnabled(v.instance.GetAnnotations()) {
+		return nil
+	}
+
+	for idx := range v.volRepPVCs {
```

Comment:
This becomes common for both Volsync and Volrep, i.e PVC labeling. @BenamarMk @youhangwang  (FYI)

Reply:
That's correct. Currently, VolSync is not using this function yet. The temporary solution in the VolSync-related CG code involves the user labeling the PVCs. We liked the idea of having users label the PVCs as it allows for multiple CGs per storageId. This feature is still missing from @youhangwang's PR. 
I think we should implement the following for both VolSync and VolRep:
1. Provide a common label CG name, such as `ramendr.openshift.io/cg-label` for a lack of a better name.
2. If the PVC contains that label, then the PVC label value for CG should be the combination of `ramendr.openshift.io/consistency-group: storageId + cg-label`.

Reply:
yes, I'm waiting for this pr merged, then I can rebase my pr to adopt this func.

Reply:
>     1. Provide a common label CG name, such as `ramendr.openshift.io/cg-label` for a lack of a better name.
> 
>     2. If the user labels the PVC with that label, then the PVC label value for CG should be the combination of `ramendr.openshift.io/consistency-group: storageId + cg-label`.

Ack! if it is required that PVCs be separated into CGs of their own, even as they are selected by a common pvcSelector provided to Ramen, this label can help create sub-cg groups as desired.

Would it be safe to assume that this would be in the future, IOW not part of this PR or the initial CG PRs. The initial set would just group all PVCs to a cg, when possible, giving no further granularity to the app owner.

Reply:
Agreed